WordPress/src/main/java/org/wordpress/android/ui/news/NewsManager.kt

```diff
@@ -0,0 +1,54 @@
+package org.wordpress.android.ui.news
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MediatorLiveData
+import org.wordpress.android.models.news.NewsItem
+import org.wordpress.android.ui.prefs.AppPrefsWrapper
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Business logic related to fetching/showing the News Card (a card used for announcing new features/updates).
+ */
+@Singleton
+class NewsManager @Inject constructor(val newsService: NewsService, val appPrefs: AppPrefsWrapper) {
+    private val dataSourceMediator: MediatorLiveData<NewsItem?> = MediatorLiveData()
+    private var dataSource: LiveData<NewsItem?> = newsService.newsItemSource()
```

Comment:
I think this (and everything in the chain) could be `LiveData<NewsItem>`, the value field is nullable by default anyway.

Reply:
Ohh, I didn't know that the value field is nullable by default:).  👍